types/mimos/index.d.ts

```diff
@@ -1,10 +1,11 @@
 // Type definitions for mimos 3.0
 // Project: https://github.com/hapijs/mimos
 // Definitions by: AJP <https://github.com/AJamesPhillips>
+//                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
-import { DataStructure as MimeDbDataStructure } from 'mime-db';
+import database = require('mime-db');
```

Comment:
This line shouldn't need to be changed, `import` still works with declaration merged exports.